### PR TITLE
Fix Accordion Deselection

### PR DESCRIPTION
**Pull request by clmntnbrn**
fixing accordian

### CODE REVIEW

File: css/accordion.css [hunk 1]
```diff
@@ -71,6 +71,13 @@ html {
   padding: 0.5rem 1.2rem;
 }
 
+.acc input:checked ~ label .rotate {
```
Comment (JoeStanton) [inline css/accordion.css:74]: Need to check browser compatibility of this selector. There might be a simpler one that fits just as well, too.

File: js/index.js [hunk 2]
```diff
@@ -1,7 +1,22 @@
-$(document).ready(function() {
+$(document).ready(() => {
```
Comment (JoeStanton) [inline js/index.js]: Switch this to vanilla JS, no need for jQuery

**Review by JoeStanton — DISMISSED** [review]

File: css/accordion.css (hunk 1 above)
Comment (clmntnbrn) [reply css/accordion.css:74]: @JoeStanton I can't! Please help 🙏 

I did manage to replace the jquery though 👏

**Comment by clmntnbrn** [comment]
@sashaward shall we merge anyway and leave the ~ problem for another time?

**Comment by sashaward** [comment]
@clmntnbrn Yeah I reckon so!!

**Comment by clmntnbrn** [comment]
@sashaward 👍 cool can you review and approve ;)

**Review by sashaward — APPROVED** [review]

**Pull request merged**